Use /usr/bin/editor instead of vi as the default editor when EDITOR and  
ROLO_EDITOR environment variables are not defined.

 -- Rafael Laboissiere <rafael@debian.org>  Sun, 13 Jul 2008 17:51:18 +0200

Index: rolo-011.dfsg.1/doc/rolo.1
===================================================================
--- rolo-011.dfsg.1.orig/doc/rolo.1	2008-07-13 17:47:54.000000000 +0200
+++ rolo-011.dfsg.1/doc/rolo.1	2008-07-13 17:48:07.000000000 +0200
@@ -41,7 +41,7 @@
 If ROLO_EDITOR is set, its value is used as the name of the program
 to invoke when displaying contact entries in advanced edit. If
 ROLO_EDITOR is not set, then EDITOR is used. If EDITOR is also not
-set, the default value of vi is used.
+set, the default value of /usr/bin/editor is used.
 .El
 .Sh FILES
 .Bl -tag -width "~/.rolo/contacts.vcf" -compact
Index: rolo-011.dfsg.1/src/main.c
===================================================================
--- rolo-011.dfsg.1.orig/src/main.c	2008-07-13 17:47:16.000000000 +0200
+++ rolo-011.dfsg.1/src/main.c	2008-07-13 17:47:41.000000000 +0200
@@ -87,7 +87,7 @@
       editor = getenv ("EDITOR");
       if (NULL == editor)
         {
-          editor = strdup ("vi");
+          editor = strdup ("/usr/bin/editor");
           return editor;
         }
     }
